docs/content/users/quickstart.md

```diff
@@ -162,6 +162,7 @@ ddev launch
 ## Drupal
 
 === "Drupal"
```

Comment:
It reads odd to have at first a tab called drupal and then tabs with specific version numbering and technically the order is 10, 11, 9, 6/7... plus it feels odd to have the explanation inside the box on just one out of the three tabs this explanation applies to. I wonder if the following would make sense. 

move ```For all versions of Drupal 8+ the Composer techniques work. The settings configuration is done differently for each Drupal version, but the project type is “drupal”.``` outside the box, between the title Drupal and the box with the drupal tabs. and in regards of the tab naming and ordering, how about switching the position between drupal 10 and 11 and renaming `drupal` to `Drupal 10` and `Drupal 11` to `Drupal 11 (dev)` (or something similar)?

Reply:
It's hard to follow your request. Maybe you can do a mockup, or just edit as you see fit and show what happens with `make mkdocs-serve`

Reply:
![Screenshot 2024-04-07 at 03 01 57](https://github.com/ddev/ddev/assets/1665422/2195b191-6d2d-405d-9c72-710a37c94ef4)

I meant to move the description out of the drupal tab and perhaps slightly adjust it to something like: 

```
For all versions of Drupal 8+ the installation is done via Composer. The settings configuration is different for every version of Drupal, but the project type remains just "drupal".
```
and changing the tab labels  to something like:  

Drupal 11 (dev), Drupal 10, Drupal 9, Drupal 6/7,  Git Clone

Reply:
It's now like this: 
<img width="748" alt="image" src="https://github.com/ddev/ddev/assets/112444/27f16ed7-7e15-464f-b961-09fddc38edee">